test/images/agnhost/openidmetadata/openidmetadata.go

```diff
@@ -169,12 +170,14 @@ func gettoken() (string, error) {
 func withExternalOAuth2Client(ctx context.Context) (context.Context, error) {
 	// Use the default http transport with the system root bundle,
 	// since it's validating against the external internet.
+	client := &http.Client{
+		Transport: http.DefaultTransport,
+	}
+	client.Transport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
disabling security when fetching the keyset doesn't seem correct

it seems like either system roots or in-cluster CA bundle should be usable for fetching the verification keys... the point of this test is supposed to be that in-cluster clients can actually use the published keyset info

Reply:
Ack, let me dig into the in-cluster CA bundle